python/smqtk/web/geospace/deploy/docker/Dockerfile

```diff
@@ -60,10 +60,7 @@ RUN mkdir -p /home/kwuser/.girder/logs
 # Memex's proxy can't use the git protocol
 #RUN git config --global url."https://".insteadOf git://
 
-RUN git clone git://github.com/memex-explorer/geoinfo_space.git && \
-    cd geoinfo_space && \
-    git checkout optimizations && \
-    git reset --hard
+RUN git clone git://github.com/memex-explorer/geoinfo_space.git
```

Comment:
Can this be changed to use the SMQTK repo? Or is that no runable given is state of recent transference?